### PR TITLE
Add information on new resolvers and conduct load usages

### PR DESCRIPTION
**Pull request by longshorej**

### CODE REVIEW

**Comment by longshorej** [comment]
[Document recent CLI enhancements](https://trello.com/c/wnhXG1ow/652-document-recent-cli-enhancements)

File: src/main/play-doc/operation/CLI.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ commands:
 ...
 ```
 
-## Packaging configuration
+## Packaging Configuration
```
Comment (markusjura) [inline src/main/play-doc/operation/CLI.md]: I'd use `Packaging configuration` instead. We usually write the headline words in lowercase as far as the word is not a specific term. I know that we are not fully consistent across the documentation but this is the general rule.

File: src/main/play-doc/operation/CLI.md [hunk 2]
```diff
@@ -55,14 +55,12 @@ cd #back to your home folder so we know where we're playing
 echo "export POLL_INTERVAL=500ms" >> ./visualizer-poll-interval.sh
 ```
 
-Once the configuration file is ready, we need to package it up to a bundle. We can use the `shazar` command provided by the CLI which zips configuration files and appends secure digest of the archive to the created bundle files.
+Once the configuration file is ready, we can load it directly into ConductR -- configuration is always provided as the second parameter to the `conduct load` command.
```
Comment (markusjura) [inline src/main/play-doc/operation/CLI.md]: Use `into ConductR. The configuration file is always`

**Review by markusjura — APPROVED** [review]
Looking good. Only some minor comments.

File: src/main/play-doc/operation/CLI.md (hunk 2 above)
Comment (longshorej) [reply src/main/play-doc/operation/CLI.md]: Fixed.

File: src/main/play-doc/operation/CLI.md (hunk 1 above)
Comment (markusjura) [reply src/main/play-doc/operation/CLI.md]: I am wrong here. Based on http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html we need to capitalize `Configuration` as well.

File: src/main/play-doc/developer/CreatingBundles.md [hunk 3]
```diff
@@ -23,13 +23,13 @@ In addition, Docker images can be run from "universal" bundles. The following se
 ## Producing an OCI bundle with Docker
 ConductR is capable of running images that have been exported from Docker. To do this, use the `bndl` tool to convert a docker image into a ConductR bundle. Behind the scenes, we're using the standard OCI Image format so you can be sure your images are free of vendor lock-in. Additionally, this approach doesn't require Docker to be installed in production; ConductR provides everything necessary to run these images.
 
-The following command will load your docker image directly into ConductR.
+The following command will resolve your docker image from the Docker registry and load it directly into ConductR.
```
Comment (edwardcallahan) [inline src/main/play-doc/developer/CreatingBundles.md]: "from the Docker registry" -> "from DockerHub" or "from the DockerHub registry", no?

Comment (longshorej) [reply src/main/play-doc/developer/CreatingBundles.md]: Good point. In addition to that we also need to consider that private repositories are supported. Perhaps "from DockerHub and load it directly into ConductR." followed by another sentence with an example on how to resolve from private registry?

Comment (edwardcallahan) [reply src/main/play-doc/developer/CreatingBundles.md]: Love it

File: src/main/play-doc/operation/CLI.md (hunk 1 above)
Comment (huntc) [reply src/main/play-doc/operation/CLI.md]: I think that being consistent is more important than being "correct" here given that being "correct" is worth talking about. :-)

Personally I'd prefer that CSS styling be the means of capitalising a heading as, after all, it is a style.

**Review by huntc — APPROVED** [review]

**Pull request merged**